examples/basic/boid_flockers/model.py

```diff
@@ -5,10 +5,11 @@
 """
 
 import numpy as np
-from agents import Boid
 
 import mesa
 
+from .agents import Boid
```

Comment:
Because imports in Python suck. Or maybe not and I am too stupid to fully understand them. Here is my understanding, but it might be a little bit off in the details.

1. `from mesa.examples.basic.boid_flockers/agents import Boid` This is an absolute import, which would work but obviously the examples are meant to be copyable, so we don't use that here
2. `from agents impoprt Boid` This is a relative import, but it imports relative to the current working directory. So its fine if you run the example in that folder, but if you run it from somewhere else it wont work
3. `from .agents import Boid` This is a relative import, relative to the location of the file in the file system

I hope that makes it clear why I choose 3). I think the main downside of relative imports is that if you move the file you of course have to adapt. Thats why most packages work with absolute imports internally.

Reply:
Just read up about it, this is indeed the correct way to do relative imports, and I agree relative imports are the obvious choice here. Thanks a lot!